src/Http/HttpServiceProvider.php

```diff
@@ -37,6 +38,14 @@ function (Application $app) {
                 );
             }
         );
+
+        $app['http.api_key_request_authorizer'] = $app->share(
+            function (Application $app) {
+                return new ApiKeyPsr7RequestAuthorizer(
+                    $app['api_key']
+                );
```

Comment:
De `ApiKeyPsr7RequestAuthorizer` kan nu normaal geen `null` values gebruiken, en zou ook wat vreemd zijn eigenlijk. Misschien kan je hier checken of de api key `null` is, zo ja dan kan je een `Passthrough` implementatie terug geven ipv de echte `ApiKeyPsr7RequestAuthorizer`? Dan moet je die `Passthrough` nog wel voorzien in `cultuurnet/udb3`.

Reply:
De `Passthrough` implementatie is dan eigenlijk een soort van `NullObject` in deze context. (Cfr. `NullLogger`)